core/templates/dev/head/editor/ExplorationEditorAdvancedFeaturesService.js

```diff
@@ -19,17 +19,24 @@
 
 oppia.factory('explorationAdvancedFeaturesService', [function() {
   var _settings = {
-    areParametersEnabled: false,
-    areGadgetsEnabled: false
+    areFallbacksEnabled: false,
```

Comment:
You might want to also put these in alphabetical order in the html if there's no reason not to.